src/vswhere/Program.cpp

```diff
@@ -12,6 +14,10 @@ void WriteLogo(_In_ const CommandArgs& args, _In_ wostream& out);
 
 int wmain(_In_ int argc, _In_ LPCWSTR argv[])
 {
+    _setmode(_fileno(stdout), _O_U16TEXT);
```

Comment:
I'd prefer you stick with STL that is used throughout the rest of the project for code consistency. See [basic_streambuf::pubimbue](http://en.cppreference.com/w/cpp/io/basic_streambuf/pubimbue) as a pointer.